### PR TITLE
Clippy fix

### PR DESCRIPTION
**Pull request by zjp-CN**

### CODE REVIEW

**Comment by zjp-CN** [comment]
Seems you forget to define a  `nightly` feature in  `nvim-api`.

File: crates/nvim-api/Cargo.toml [hunk 1]
```diff
@@ -12,3 +12,6 @@ derive_builder = "0.11"
 serde = { version = "1.0", features = ["derive"] }
 serde_repr = "0.1"
 thiserror = "1.0"
+
+[features]
+nightly = []
```
Comment (noib3) [inline crates/nvim-api/Cargo.toml:17]: nit: could you move the `[features]` section before the `[dependencies]`?

Comment (noib3) [reply crates/nvim-api/Cargo.toml:17]: also, the `nightly` feature on `nvim-api` should enable `nightly` on `nvim-types`.

**Comment by zjp-CN** [comment]
For `api::autocmd::exec_autocmds` test, stderr is one line, and [`lines.len() - 2`](https://github.com/noib3/nvim-oxi/blob/86280aa35bc919334d227bc722c7ca92a3279cec/crates/oxi-test/src/lib.rs#L123) causes `thread 'main' panicked at 'attempt to subtract with overflow', src/api/autocmd.rs:84:1`. We can insert a print to see what's the stderr:

```diff
+println!("\n\n{:?}\nlines: {}\n", stderr, stderr.lines().count());
if !stderr.is_empty() {


+"LuaJIT ASSERT lj_api.c:895: lua_next: stack slot -2 is not a table\n"
+lines: 1
```

So we need an error handling to keep the stderr info when something goes wrong?

**Comment by noib3** [comment]
> So we need an error handling to keep the stderr info when something goes wrong?

Or just check how many lines stderr takes before subtracting 2. Anyway that's unrelated to this PR, I'm merging this now.

**Pull request merged**